docs/token-contracts/vesting.md

```diff
@@ -0,0 +1,346 @@
+---
+id: vesting
+title: Getting started with the Vesting contract
+sidebar_label: Vesting Contract Quick Start
+---
+
+# Introduction
+
+A _money stream_ is a series of payments at regular intervals, such as a salary,
+subscription, or other contractual disbursement.
+
+The [_Vesting Contract_](https://github.com/tqtezos/vesting-contract) represents
+an on-chain stream of Tez to a fixed address: the amount of available funds is
+updated each second and anyone may vest available funds to the target address.
+
+An example of the Vesting Contract may be found [on Edo2Net](https://better-call.dev/edo2net/KT1VdxSaoxxwBb7s9ZWDCbvR4NN54qfkwE1Q/operations).
+
+## Use Cases
+
+- Salaries
+- Grants
+  * Fund sub-projects over time
+- Subscriptions
+  * Content creation
+  * Rent
+  * Parking
+  * CDPs
+- Contracts
+  * Consultancies
+  * Vendors
+- "Reversible" fundraising: introduced [at Devcon4 by @frozeman](https://cryptoindia.io/lukso-ama-recap/),
+  investors' funds are held in streaming contracts and returned as the project evolves.
+
+# Vesting Contract
+
+The Vesting Contract holds Tez and periodically allows some of them to be
+transferred to a fixed address, i.e. "_vested_":
+
+```haskell
+parameter (or (option %setDelegate key_hash)
+              (nat %vest))
+```
```

Comment:
there is no `%default`  `unit` entrypoint?

one must delegate to a baker that doesn't share rewards then!

Reply:
I added a note and link to a LIGO wrapper that could be used to add entrypoints